security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -0,0 +1,75 @@
+//OpenShift Compliance Operator Release Notes
+[id="compliance-operator-release-notes"]
+= Compliance Operator release notes
+:context: compliance-operator-release-notes-v0
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+[id="compliance-operator-inclusive-language"]
+== Making open source more inclusive
+
+Red Hat is committed to replacing problematic language in our code, documentation, and web properties. We are beginning with these four terms: master, slave, blacklist, and whitelist. Because of the enormity of this endeavor, these changes will be implemented gradually over several upcoming releases. For more details, see link:https://www.redhat.com/en/blog/making-open-source-more-inclusive-eradicating-problematic-language[Red Hat CTO Chris Wright's message].
+
+[id="compliance-operator-release-notes-0-1-44"]
+== OpenShift Compliance Operator 0.1.44
+
+The following advisory is available for the OpenShift Compliance Operator 0.1.44:
+
+* link:https://access.redhat.com/errata/RHBA-2021:4530[RHBA-2021:4530 OpenShift Compliance Operator Bug Fix and Enhancement Update]
+
+[id="compliance-operator-0-1-44-new-features-and-enhancements"]
+=== New features and enhancements
+
+* In this release, the `strictNodeScan` option is now added to the `ComplianceScan`, `ComplianceSuite` and `ScanSetting` CRs. This option defaults to "true" which matches the previous behavior where an error occurred if a scan was not able to be scheduled on a node. Setting the option to "false" (meaning not strict, permissive) allows the Compliance Operator to be more lenient about scheduling scans. Environments with ephemeral nodes can set `strictNodeScan` to false which will allow a compliance scan to proceed, even if some of the nodes in the cluster are not available for scheduling.
```

Comment:
You might want to reconsider the use of parentheses. The ISG suggests that one avoid using them in running text because it means that the text they contain is not very important.

```suggestion
* In this release, the `strictNodeScan` option is now added to the `ComplianceScan`, `ComplianceSuite` and `ScanSetting` CRs. This option defaults to `true`, which matches the previous behavior, where an error occurred if a scan was not able to be scheduled on a node. Setting the option to `false` (meaning not strict, permissive) allows the Compliance Operator to be more lenient about scheduling scans. Environments with ephemeral nodes can set the `strictNodeScan` value to false, which allows a compliance scan to proceed even if some of the nodes in the cluster are not available for scheduling.
```